libsplinter/src/rest_api/mod.rs

```diff
@@ -380,6 +380,29 @@ pub fn percent_encode_filter_query(input: &str) -> String {
     percent_encoding::utf8_percent_encode(input, QUERY_ENCODE_SET).to_string()
 }
 
+pub fn get_header(header_key: &str, request: &HttpRequest) -> Result<String, RequestError> {
```

Comment:
Would suggest naming this `require_header`, as it fails if the header is missing.

Reply:
Addressed